### PR TITLE
Start removing jQuery

### PR DESCRIPTION
**Pull request by rwwagner90**
Some of this might be a bit tricky, but I wanted to take a first pass at it. Let me know your thoughts 😄

### CODE REVIEW

**Comment by rwwagner90** [comment]
cc @fotinakis

File: addon/snapshot.js [hunk 1]
```diff
@@ -95,26 +94,26 @@ export function percySnapshot(name, options) {
   let scope = options.scope;
 
   // Create a full-page DOM snapshot from the current testing page.
-  let domCopy = percyJQuery('html').clone();
-  let bodyCopy = domCopy.find('body');
-  let testingContainer = domCopy.find('#ember-testing');
+  let domCopy = document.documentElement.cloneNode(true);
```
Comment (fotinakis) [inline addon/snapshot.js]: @rwwagner90 This is the big one that worries me, I don't have a good sense for what jQuery.clone is doing underneath and we need these things especially to be fully backwards compatible.

This apparently is what jQuery is doing underneath:
https://j11y.io/jquery/#v=1.11.2&fn=jQuery.clone

Comment (rwwagner90) [reply addon/snapshot.js]: @fotinakis according to http://youmightnotneedjquery.com/ `$(el).clone();` is the same as `el.cloneNode(true);`

**Comment by fotinakis** [comment]
Per discussion, going to close this for now and leave things as is. We have some designs for a future version that will supersede this so we're going to invest effort there.

**Comment by rwwagner90** [comment]
No problem @fotinakis. If you need help with this future version, feel free to reach out!

**Pull request closed without merging**